packages/paste-core/components/combobox/src/multiselect/MultiselectCombobox.tsx

```diff
@@ -11,6 +11,8 @@ import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
 import {FormPillGroup, FormPill, useFormPillState} from '@twilio-paste/form-pill-group';
 import {useComboboxPrimitive, useMultiSelectPrimitive} from '@twilio-paste/combobox-primitive';
 import {InputBox, InputChevronWrapper, getInputChevronIconColor} from '@twilio-paste/input-box';
+import {Portal} from '@twilio-paste/reakit-library';
+import {useRect} from '@radix-ui/react-use-rect';
```

Comment:
question (non-blocking): Is this hook something we can copy over to paste-utils/hooks so we can own it moving forward and re-use it in packages more easily (similar to paste-libraries, but for hooks)

Reply:
We definitely could.. but then this hook is dependent on a radix/rect, so would I copy that over too? also if we copy it over we have to write tests for it.